index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>S.P.A. Quiz</title>
+  
+  <!-- Latest compiled and minified CSS -->
+  <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+<!-- Nav bar -->
+  <nav class="navbar-form navbar-fixed">
+    <input type="text" id="inputBar" class="form-control" placeholder="Search">
+    <button type="submit" class="btn btn-default">Submit</button>
+  </nav> 
+
+<!-- Put this into DOM string dynamically -->
+  <div class ="container">
+    <div id = "output" class="row">
+      
+    </div>   
+  </div>
+
+  <script type="text/javascript" src="carLot.js"></script>
+  <script type="text/javascript" src="quiz.js"></script>
+  <script type="text/javascript" src="activateEvents.js"></script>
+  <script type="text/javascript" src="finalIIFE.js"></script>
```

Comment:
This is not a great name. What if you expand your code and need another iife?  will it be final-finalIIFE?